src/app/cube/shared/learning-object/learning-object.component.ts

```diff
@@ -130,16 +132,29 @@ export class LearningObjectListingComponent implements OnInit, OnChanges, OnDest
    * @param organization string of the users affiliated organization
    * @returns string unformated or title cased
    */
-     organizationFormat(organization: string) {
-      if ( organization && (organization.charAt(1) === organization.charAt(1).toUpperCase()) ) {
-        return organization;
-      } else if (organization) {
-        return titleCase(organization);
-      }
+  organizationFormat(organization: string) {
+    if (organization && (organization.charAt(1) === organization.charAt(1).toUpperCase())) {
+      return organization;
+    } else if (organization) {
+      return titleCase(organization);
     }
+  }
```

Comment:
```suggestion
  organizationFormat(organization: string) {
    return titleCase(organization);
  }
```
Does this do the same thing?

Reply:
One was checking if it was in all caps and then just returning it. and then title casing everything else, but it does work as a one-liner. I went ahead and changed it.